mapie/tests/test_regression.py

```diff
@@ -403,16 +411,17 @@ def test_invalid_aggregate_all() -> None:
         aggregate_all(None, X)
 
 
-def test_invalid_aggregate_mask() -> None:
+def test_aggregate_with_mask_with_prefit() -> None:
     """
-    Test that wrong aggregation in MAPIE raise errors.
+    Test ``aggregate_with_mask`` in case ``cv`` is "prefit".
```

Comment:
```suggestion
    Test ``aggregate_with_mask`` in case ``cv`` is ``"prefit"``.
```

Reply:
I fear I migth not get the exact suggestion

Reply:
Double backticks needed for sphinx rendering

Reply:
Done, I think